packages/suite/src/reducers/wallet/sendFormReducer.ts

```diff
@@ -1,84 +1,85 @@
-                    // Deep-cloning to prevent buggy interaction between react-hook-form and immer, see https://github.com/orgs/react-hook-form/discussions/3715#discussioncomment-2151458
-                    // Otherwise, whenever the outputs fieldArray is updated after the form draft or precomposedForm is saved, there is na error:
-                    // TypeError: Cannot assign to read only property of object '#<Object>'
-                    // This might not be necessary in the future when the dependencies are upgraded.
-                    draft.precomposedForm = cloneObject(action.payload.formValues);
-                } else {
-                    delete draft.precomposedTx;
-                    delete draft.precomposedForm;
-                }
-                break;
-            case SEND.REQUEST_PUSH_TRANSACTION:
-                if (action.payload) {
-                    draft.signedTx = action.payload;
-                } else {
-                    delete draft.signedTx;
-                }
-                break;
-            case SEND.SEND_RAW:
-                draft.sendRaw = action.payload;
-                break;
-            case SEND.DISPOSE:
-                delete draft.sendRaw;
-                delete draft.precomposedTx;
-                delete draft.precomposedForm;
-                delete draft.signedTx;
-                break;
-            // no default
-        }
-    });
+                state.precomposedForm = cloneObject(formState);
+            },
+        )
+        .addCase(sendFormActions.storeSignedTransaction, (state, { payload: signedTx }) => {
+            state.signedTx = signedTx;
+        })
+        .addCase(sendFormActions.discardTransaction, state => {
+            delete state.precomposedTx;
+            delete state.precomposedForm;
+            delete state.signedTx;
+        })
+        .addCase(sendFormActions.sendRaw, (state, { payload: { sendRaw } }) => {
+            state.sendRaw = sendRaw;
+        })
+        .addCase(sendFormActions.dispose, state => {
+            delete state.sendRaw;
+            delete state.precomposedTx;
+            delete state.precomposedForm;
+            delete state.signedTx;
+        })
+        .addCase(extra.actionTypes.storageLoad, extra.reducers.storageLoadFormDrafts)
+        .addCase(accountsActions.removeAccount, (state, { payload }) => {
+            payload.forEach(account => {
+                delete state.drafts[account.key];
+            });
+        });
+});
+
+export const selectSendPrecomposedTx = (state: SendRootState) => state.wallet.send.precomposedTx;
+export const selectSendSignedTx = (state: SendRootState) => state.wallet.send.signedTx;
+export const selectPrecomposedSendForm = (state: SendRootState) =>
+    state.wallet.send.precomposedForm;
+export const selectSendFormDrafts = (state: SendRootState) => state.wallet.send.drafts;
 
-export default sendFormReducer;
+export default prepareSendFormReducer;
```

Comment:
No default exports please, just export the `prepareSendFormReducer` as named export

Reply:
fixed by b2b40263ba